alpharaw/ms_data_base.py

```diff
@@ -128,13 +158,29 @@ def _load_meta_from_hdf(self, hdf: HDF_File):
         self.instrument = hdf.ms_data.meta.instrument
 
     def save_hdf(self, _path: str):
+        """
+        Save data into HDF5 file
+
+        Parameters
+        ----------
+        _path : str
+            HDF5 file path
```

Comment:
in the past I learned that there's always a huge confusion in naming around files and paths.
So I made it a habit to be very precise, e.g. on `absolute` vs `relative` (also on `file` vs `file_name`)
Maybe rephrase to "absolute path to HDF5 file"